_CodingChallenges/071-minesweeper.md

```diff
@@ -112,7 +112,6 @@ contributions:
     author:
       name: "Loïc Rico"
       url: https://github.com/ricoloic/
-    video_id: m4FEp0jb0RA
```

Comment:
can't we add a video_id to the additions mades ?

Reply:
@ricoloic either the URL or a video can be specified, unfortunately. Do you want the video instead of the website?

Reply:
@gruselhaus haaa ok sorry didn't notice it was one or the orther well you can leave it as is, thank a lot